openldap/osfamilymap.yaml

```diff
@@ -16,7 +16,6 @@ Debian:
   server_config: /etc/ldap/slapd.conf
   server_defaults: /etc/default/slapd
   server_pkg: slapd
-  database: mdb
```

Comment:
This I can see here: https://github.com/saltstack-formulas/openldap-formula/blob/9f758a4a599bb521b2b40cc9b4eab249c53bb4ec/openldap/defaults.yaml#L10.